BatteryReporter.cs

```diff
@@ -0,0 +1,29 @@
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+
+namespace BatteryCharging
+{
+    public interface IReporter
+    {
+        void DisplayBatteryStatus(bool IsBatteryFaulty, List<Tuple<string, string>> batteryReports);
+    }
+    public class BatteryReporter : IReporter
```

Comment:
BatteryReporter class name is  too Generalized

Reply:
Class name is modified to BatteryMeasuresReporter